native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java

```diff
@@ -208,6 +208,18 @@ private void removeListeners() {
             this.removeTextChangedListener(mTextWatcher);
         }
 
+        private boolean isScrollBySystem(int bottom) {
```

Comment:
isScrollBySystem -> isScrolledBySystem

Reply:
May be can just more meaningful function name, such as isSystemAdjustScreenWhenPopKeyboard.